src/Zend/Session.php

```diff
@@ -47,6 +47,17 @@ class Zend_Session extends Zend_Session_Abstract
      */
     protected static $_throwStartupExceptions = true;
 
+
+    /**
+     * Whether or not session write warnings should be suppressed
+     *
+     * If current process gets in a race condition with another php process,
+     * PHP throws a "Failed to write session data using user defined save handler" warning if the session
+     *
+     * @var bool
+     */
+    protected static $_suppressSessionWriteWarning = true;
```

Comment:
I think that this should be an opt-in option, as to not change the library's current behavior. Is it too much work in your implementation to enable this option rather than relying on it to be enabled?